openwisp-config/files/openwisp.agent

```diff
@@ -491,7 +491,7 @@ apply_configuration() {
 	# store unmanaged config (if enabled)
 	call_store_unmanaged
 	# update local configuration
-	/usr/sbin/openwisp-update-config --conf="$1" --merge="$MERGE_CONFIG"
+	/usr/sbin/openwisp-update-config --conf="$1" --merge="$MERGE_CONFIG" 2>&1 | logger -t openwisp -p daemon.err -s
```

Comment:
I added some bogus code in openwisp-update-config on purpose, which is now exiting with 1:

```
openwisp-update-config
lua: /usr/sbin/openwisp-update-config:15: attempt to call global 'PLEASERROR' (a nil value)
stack traceback:
	/usr/sbin/openwisp-update-config:15: in main chunk
	[C]: ?
```

Then this happens:

```
Wed Jul  6 23:23:16 2022 daemon.info openwisp: Local configuration outdated
Wed Jul  6 23:23:16 2022 daemon.info openwisp: Downloading configuration from controller...
Wed Jul  6 23:23:17 2022 daemon.info openwisp: Configuration downloaded, now applying it...
Wed Jul  6 23:23:17 2022 daemon.err openwisp: lua: /usr/sbin/openwisp-update-config:15: attempt to call global 'PLEASERROR' (a nil value)
Wed Jul  6 23:23:17 2022 daemon.err openwisp: stack traceback:
Wed Jul  6 23:23:17 2022 daemon.err openwisp: 	/usr/sbin/openwisp-update-config:15: in main chunk
Wed Jul  6 23:23:17 2022 daemon.err openwisp: 	[C]: ?
Wed Jul  6 23:23:22 2022 daemon.info openwisp: Testing configuration...
Wed Jul  6 23:23:23 2022 daemon.info openwisp: Configuration test succeeded
Wed Jul  6 23:23:23 2022 daemon.info openwisp: Configuration applied successfully
```

Configuration applied successfully?